flutter_vlc_player_platform_interface/lib/src/enums/subtitles/vlc_subtitle_color.dart

```diff
@@ -0,0 +1,40 @@
+enum VlcSubtitleColor {
+  Black,
+  Gray,
+  Silver,
+  White,
+  Maroon,
+  Red,
+  Fuchsia,
+  Yellow,
+  Olive,
+  Green,
+  Teal,
+  Lime,
+  Purple,
+  Navy,
+  Blue,
+  Aqua,
+}
+
+extension VlcSubtitleColorExtensionMap on VlcSubtitleColor {
+  static const valueMap = {
+    VlcSubtitleColor.Black: 0,
+    VlcSubtitleColor.Gray: 8421504,
+    VlcSubtitleColor.Silver: 12632256,
+    VlcSubtitleColor.White: 16777215,
+    VlcSubtitleColor.Maroon: 8388608,
+    VlcSubtitleColor.Red: 16711680,
+    VlcSubtitleColor.Fuchsia: 16711935,
+    VlcSubtitleColor.Yellow: 16776960,
+    VlcSubtitleColor.Olive: 8421376,
+    VlcSubtitleColor.Green: 32768,
+    VlcSubtitleColor.Teal: 32896,
+    VlcSubtitleColor.Lime: 65280,
+    VlcSubtitleColor.Purple: 8388736,
+    VlcSubtitleColor.Navy: 128,
+    VlcSubtitleColor.Blue: 255,
+    VlcSubtitleColor.Aqua: 65535,
```

Comment:
Let's use Effective Dart naming convention for enums: https://dart-lang.github.io/linter/lints/constant_identifier_names.html
```suggestion
enum VlcSubtitleColor {
  black,
  gray,
  silver,
  white,
  maroon,
  red,
  fuchsia,
  yellow,
  olive,
  green,
  teal,
  lime,
  purple,
  navy,
  blue,
  aqua,
}

extension VlcSubtitleColorExtensionMap on VlcSubtitleColor {
  static const valueMap = {
    VlcSubtitleColor.black: 0,
    VlcSubtitleColor.gray: 8421504,
    VlcSubtitleColor.silver: 12632256,
    VlcSubtitleColor.white: 16777215,
    VlcSubtitleColor.maroon: 8388608,
    VlcSubtitleColor.red: 16711680,
    VlcSubtitleColor.fuchsia: 16711935,
    VlcSubtitleColor.yellow: 16776960,
    VlcSubtitleColor.olive: 8421376,
    VlcSubtitleColor.green: 32768,
    VlcSubtitleColor.teal: 32896,
    VlcSubtitleColor.lime: 65280,
    VlcSubtitleColor.purple: 8388736,
    VlcSubtitleColor.navy: 128,
    VlcSubtitleColor.blue: 255,
    VlcSubtitleColor.aqua: 65535,
```